grazer/src/main/webapp/WEB-INF/spring/root-context.xml

```diff
@@ -13,8 +14,21 @@
 	http://www.springframework.org/schema/aop http://www.springframework.org/schema/aop/spring-aop-3.2.xsd
 	http://www.springframework.org/schema/task http://www.springframework.org/schema/task/spring-task-3.0.xsd
 	http://www.springframework.org/schema/cache http://www.springframework.org/schema/cache/spring-cache.xsd
+	http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx.xsd
 	">
 
+	<mvc:annotation-driven />
+	<context:component-scan base-package="edu.asu.diging.grazer" />
```

Comment:
why are you scanning all packages? there is already a component-scan tag, is that not enough?